e2e/specs/deploy_button.spec.js

```diff
@@ -14,21 +14,36 @@
  * limitations under the License.
  */
 
-describe("displays deploy button correctly", () => {
-  before(() => {
+describe("deploy button and modal", () => {
+  beforeEach(() => {
     cy.loadApp("http://localhost:3000/?_stcore_testing=true");
+
+    cy.prepForElementSnapshots();
   });
 
-  it("deploy button matches the snapshot", () => {
+  it("renders the deploy button correctly", () => {
     cy.get("div[class='stDeployButton']").matchThemedSnapshots(
       "deploy_button"
     );
   });
 
-  it("deploy dialog matches the snapshot", () => {
-    cy.get("div[class='stDeployButton'] > button").click({force: true})
+  it("renders the light deploy dialog correctly", () => {
+    cy.changeTheme("Light");
```

Comment:
Nit: since light theme is the default, this line just slows down the test run - can be removed

Reply:
Gotcha! Fixed it on [382d0f9](https://github.com/streamlit/streamlit/pull/6688/commits/382d0f90e12b04e4f07fe948594fc0f8e1de9d63)